src/Internal/Logger.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * Class Logger
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal;
+
+use Exception;
+use WC_Logger;
+use WC_Log_Levels;
+use WC_Payment_Gateway_WCPay;
+use WCPay\Core\Mode;
+
+defined( 'ABSPATH' ) || exit; // block direct access.
```

Comment:
We don't need those lines in `src` classes: It would be a blank screen anyway.

Reply:
Removed